src/main/java/org/embulk/output/elasticsearch/ElasticsearchHttpClient.java

```diff
@@ -48,6 +48,7 @@
     // @see https://github.com/elastic/elasticsearch/blob/master/core/src/main/java/org/elasticsearch/cluster/metadata/MetaDataCreateIndexService.java#L108
     private final long maxIndexNameBytes = 255;
     private final List<Character> inalidIndexCharaters = Arrays.asList('\\', '/', '*', '?', '"', '<', '>', '|', '#', ' ', ',');
+    private final long maxSnapshotWaitingTime = 1800;
```

Comment:
I changed a littlea 80c72a
I don't have enough confident for this value.
It should depends on Es server's specs and repository types(S3/Local FileSystem, HDFS...). Then to make configurable is one idea.
As one of the guideline, Elastic Cloud capture snapshot at every 30 minutes with S3 repository.